dist/reporttable.js

```diff
@@ -169,6 +169,7 @@
 
         this.container.before(this.header);
         this.container.after(this.footer);
+        this.container = this.container.parents(".koddiTableGraphContainer");
```

Comment:
`.koddiTableGraphContainer` is a part of Koddi core, so not really something this repo should know about. I'm not sure why the container is being changed here. Whatever we pass in to the container option should be what we need to use. Not sure what this solves, but can we make it work without this piece?